service/awsconfig/v5/resource/cloudformation/current.go

```diff
@@ -84,16 +84,24 @@ func (r *Resource) GetCurrentState(ctx context.Context, obj interface{}) (interf
 	if err != nil {
 		return StackState{}, microerror.Mask(err)
 	}
+	versionBundleVersion, err := getStackOutputValue(outputs, versionBundleVersionOutputKey)
+	if err != nil {
+		return StackState{}, microerror.Mask(err)
+	}
 
 	outputStackState := StackState{
-		Name:                     stackName,
+		Name: stackName,
+
 		MasterImageID:            masterImageID,
 		MasterInstanceType:       masterInstanceType,
 		MasterCloudConfigVersion: masterCloudConfigVersion,
+
 		WorkerCount:              workers,
 		WorkerImageID:            workerImageID,
 		WorkerInstanceType:       workerInstanceType,
 		WorkerCloudConfigVersion: workerCloudConfigVersion,
+
+		VersionBundleVersion: versionBundleVersion,
```

Comment:
The version bundle version magic is also here. We fetch the current version bundle version fro the stack we fetched from AWS. 

Let me know when you want to see a separate PR. I am still exploring and learning by doing so I cannot separate in the first place as I would like to in hindsight.